tests/helpme/test_create.py

```diff
@@ -343,3 +343,79 @@ def test_determinant_spline(
             det_spl.evaluate(queries=rnd_queries).ravel(),
             np.linalg.det(sp_i.jacobian(queries=rnd_queries)),
         ), f"{sp_i.whatami} at index {idx} failed determinant spline"
+
+
+def test_swept_basic_functionality():
```

Comment:
can you be more specific? you are just checking number of cps

Reply:
Did it, I guess